server/src/controllers/userRoomMapping/index.ts

```diff
@@ -29,7 +29,7 @@ export function getRoomsOfUser(req: Request, res: Response) {
 }
 
 export function getUsersOfRoom(req: Request, res: Response) {
-  const { roomId } = req.body
+  const roomId = req.query.roomId.toString()
```

Comment:
Same problem here. `req.query.roomId` may be undefined. Calling `toString()` on it would result in an exception, crashing the server. Probably should use optional chaining and then check the final value of `roomId`